### ISSUE TITLE
tkg.io down

### ISSUE THREAD

**Issue by asgeirtj** [kairyu/tkg#35]
is it possible to run this myself?

**Comment by cemegginson** [comment]
Yes. You can download the repo as a zip file and extract it. Then just open index.html and your browser will open it.

**Comment by asgeirtj** [comment]
Haven't managed it to get it to work because the Chrome app deprecation thing
